docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart" %}
+	 <input type="submit" value="clear" {{id}}__clear" />
```

Comment:
A stray `{{id}}__clear"` has snuck in here I think

Reply:
Thanks for pointing this out @paulina-a!

I thought we needed this, so the form knows which cart we need to clear?

Reply:
No problem. Each customer can only have 1 cart so it will just clear out everything in their cart.

Reply:
Cool, thanks for confirming 🙌